### PR TITLE
Remove superfluous dependencies

### PR DESCRIPTION
**Pull request by ChrisLovering**
## Description

1. Change AoC helpers to use arrow over pytz 
2. Remove bs4 and pytz 
3. Update arrow to 1.1.0

pytz is only used in one file in the whole bot, so we can remove the dependency by changing AoC to use arrow.  
I chose to use `America/Chicago` arbitrarily, for no other reason that being the first tz google returned that was in EST.

Beautifulsoup4 is no longer used, it used to be used in AoC code a while ago.

I also updated arrow to 1.1.0 while I was here as the breaking changes don't impact us.

## Did you:

- [x] Join the [**Python Discord Community**](https://discord.gg/python)?
- [x] Read all the comments in this template?
- [x] Ensure there is an issue open, or link relevant discord discussions?
- [x] Read the [contributing guidelines](https://pythondiscord.com/pages/contributing/contributing-guidelines/)?

### CODE REVIEW

**Review by ToxicKidz — CHANGES_REQUESTED** [review]
I get this error when I run `.aoc countdown`
```py
05/17/21 18:41:12 - bot.exts.evergreen.error_handler ERROR: Unhandled command error: tzinfo argument must be None or of a tzinfo subclass, not type 'str'
Traceback (most recent call last):
  File "python3.9\lib\site-packages\discord\ext\commands\core.py", line 85, in wrapped 
    ret = await coro(*args, **kwargs)
  File "~\sir-lancebot\bot\exts\christmas\advent_of_code\_cog.py", line 103, in aoc_countdown
    datetime_now = datetime.now(_helpers.EST)
TypeError: tzinfo argument must be None or of a tzinfo subclass, not type 'str'
```

**Review by ToxicKidz — APPROVED** [review]
:shipit:

**Comment by jb3** [comment]
> I chose to use America/Chicago arbitrarily, for no other reason that being the first tz google returned that was in EST.

Will this be consistent? Arrow will presumably implement daylight savings for actual locations.

File: bot/exts/christmas/advent_of_code/_helpers.py [hunk 1]
```diff
@@ -48,7 +48,7 @@
 )
 
 # Create an easy constant for the EST timezone
-EST = pytz.timezone("EST")
+EST = "America/Chicago"
```
Comment (Shivansh-007) [inline bot/exts/christmas/advent_of_code/_helpers.py:51]: According to @janine9vn 
> Chicago is CST/CDT. So one hour before EST/EDT

[New York](https://www.zeitverschiebung.net/en/timezone/america--new_york), should be the right timezone.

**Comment by ChrisLovering** [comment]
> > I chose to use America/Chicago arbitrarily, for no other reason that being the first tz google returned that was in EST.
> 
> Will this be consistent? Arrow will presumably implement daylight savings for actual locations.

Arrow will shift for daylight savings, which I assume is what we want, rather than a static GMT offset.

**Review by Shivansh-007 — APPROVED** [review]
Looks Good To Me! :shipit:

**Review by jb3 — APPROVED** [review]

**Pull request merged**